test/e2e_node/hugepages_test.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
Are we now restricting this to be a linux only e2e test? Was that previously the case?

Reply:
Ahh. Good point! I think this was because i import `github.com/opencontainers/runc/libcontainer/cgroups` and use a variable only available in the linux-code (so bazel complained). `hugetlb` is linux-only, but it may be a point allow the test to run on non-linux too? :thinking:

These e2e-node tests only run on linux:
```
util_xfs_linux.go                                                                                                                      
1:// +build linux                                                                                                                      

density_test.go
1:// +build linux

e2e_node_suite_test.go
1:// +build linux

container_manager_test.go
1:// +build linux

benchmark_util.go
1:// +build linux

restart_test.go
1:// +build linux

node_container_manager_test.go
1:// +build linux

resource_collector.go
1:// +build linux

resource_usage_test.go
1:// +build linux
```

Reply:
Gotcha - ok! I'm a little surprised that this test ran on non-linux before, but I don't think its a problem restricting it to linux in the future.